tests/vmi_multus_test.go

```diff
@@ -464,10 +464,12 @@ var _ = Describe("[Serial]Multus", func() {
 					&expect.BSnd{S: fmt.Sprintf("ip addr show eth0 | grep %s | wc -l", interfacesByName["default"].MAC)},
 					&expect.BExp{R: "1"},
 				}, 15)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
@qinqon since you already work on multus: This fix here reveals that we actually get an error here, which we probably don't want.

Reply:
With the correct check pull-kubevirt-e2e-k8s-cnao-1.19 is now consistently failing.

/hold

Holding until resolved.

Reply:
@zhouhao3 maybe you can revert your fix in this file and this can be dealt with in a separate PR?

Reply:
> @zhouhao3 maybe you can revert your fix in this file and this can be dealt with in a separate PR?

+1.

I'll help debug that.

Reply:
Looking at the prow job, the command is `ip addr show eth0 | grep 52:54:00:5f:3d:a3 | wc -l` and the interface has 
```yaml
interfaces": [                                                                  
                    {                                                           
                        "ipAddress": "10.244.196.162",                          
                        "mac": "52:54:00:5f:3d:a3",                             
                        "name": "default",                                      
                        "ipAddresses": [                                        
                            "10.244.196.162",                                   
                            "fd10:244::c4a1"                                    
                        ]                                                       
                    },                                                          
                    {                                                           
                        "mac": "8a:60:8f:8d:e1:a6",                             
                        "name": "linux-bridge"                                  
                    }                                                           
~                                                          
```

Reply:
Although one problem we can have is capital vs non capital, in case ip addr show eth0 show it with capital letters.

Reply:
@maiqueb let me know if you need some help.

Reply:
From what I understand, the command is timing out when executed on the VM; no clue as to why.

Pretty easy to reproduce locally.

Reply:
Also we can use `SafeExpectBatchWithResponse` so we check the output at different expect
```
CheckCloudInitMetaData = func(vmi *v1.VirtualMachineInstance, testFile, testData string) {
            cmdCheck := "cat /mnt/" + testFile + "\n"                           
            res, err := console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
                &expect.BSnd{S: "sudo su -\n"},                                 
                &expect.BExp{R: console.PromptExpression},                      
                &expect.BSnd{S: cmdCheck},                                      
                &expect.BExp{R: testData},                                      
            }, 15)                                                              
            if err != nil {                                                     
                Expect(res[1].Output).To(ContainSubstring(testData))            
            }                                                                   
        }        
```

Reply:
OK found it; `\n` is missing at the end of the command.

https://github.com/kubevirt/kubevirt/pull/4923 fixes this. I recommend dropping this commit, and rebase on top of it.

Reply:
Done.